subsys/bluetooth/host/gatt.c

```diff
@@ -1813,11 +1972,11 @@ static u8_t update_ccc(const struct bt_gatt_attr *attr, void *user_data)
 
 		if (ccc->cfg[i].value) {
 			gatt_ccc_changed(attr, ccc);
-#if defined(CONFIG_BT_GATT_DYNAMIC_DB)
-			if (ccc->cfg_changed == sc_ccc_cfg_changed) {
-				sc_restore(&ccc->cfg[i]);
+			if (IS_ENABLED(CONFIG_BT_GATT_SERVICE_CHANGED)) {
```

Comment:
Merge the nested if-statements with && ?

Reply:
Done in latest push!